custom/icds_reports/utils/aggregation_helpers/distributed/gov_vhnd_forms.py

```diff
@@ -0,0 +1,56 @@
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_GOV_VHND_TABLE
+from custom.icds_reports.utils.aggregation_helpers import month_formatter
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import (
+    StateBasedAggregationPartitionedHelper,
+)
+
+
+class GovVhndFormAggDistributedHelper(StateBasedAggregationPartitionedHelper):
```

Comment:
I think despite of me saying to create a separate table and aggregate it. I think it may make sense to just add this aggregation as a part of `AggGovDashboardHelper` and introduce all-new columns in that table. We will gain the following benefits from it.
1) Since now are already aggregating gov dashboard separately, it does not make sense to let this slow down the standard aggregation when we can make this part of gov dashboard aggregation.
2) All gov dashboard related info will be in a single table which will be isolated from standard dashboard.

Reply:
It will also reduce your code change size. And size of the Gov_dashboard table is significantly smaller than agg_awc so its faster to query from that.

Reply:
But my main moto is not to slow down the standard aggregation whenever possible.

Reply:
makes sense.Should i change this?

Reply:
yes please.

Reply:
done